remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerInterceptor.java

```diff
@@ -127,7 +126,7 @@ public Object visitPutKeyValueCommand(InvocationContext ctx, PutKeyValueCommand
                   .addProtoFile(key, (String) command.getValue());
             try {
                serializationContext.registerProtoFiles(source);
-            } catch (IOException | DescriptorParserException e) {
```

Comment:
This change in protostream 4.3 (IOException no longer declared to be thrown) is regrettably not backward compatible with 4.2.x, so I will undo it in next 4.3.x alpha release. For now, keep this as is.

Reply:
> 13.4.21. Method and Constructor Throws
>
> Changes to the throws clause of methods or constructors do not break compatibility with pre-existing binaries; these clauses are checked only at compile time.

Reply:
Was not worried about binary compat here. This change breaks compilation.